### PR TITLE
Show DAP-G on Y-axis

### PR DESCRIPTION
**Pull request by amkwong**
## Ticket
n/a

## Purpose
Add an additional field (Posterior Inclusion Probability - PIP, results from Wen et al.'s DAP-G analysis of GTEx v8) to be plotted on the Y-axis (where the data is available). This requires a new database file containing the additional data.

Since the additional data is somewhat sparse, my approach is to retrieve the data for a given query, convert this into a Python dictionary, pass that dictionary to the parser, and add the PIP values to the points if possible. This approach should broadly work well enough for any additional data we may want to add.

## How to test this
View the single-variant view for the test variant chr19:488506, and click the "PIP" button to ensure the correct variable is plotted on the Y-axis. Switch back and forth between other Y-axis variables, and other buttons, to confirm that it works (and that I didn't break something else). 

### Adds new data
Download the test database file (data/GTEx_v8_finemapping_DAPG.sqlite.db) to your current data directory (as specified in your local settings) - please note that this test file contains only information for chr19:488506. For all available data, you can use the full database (~3GB), which you can find at the following location: /net/amd/amkwong/browseQTL/data/GTEx_v8_finemapping_DAPG.sqlite.db

### Modifies dependencies
Be sure to update python dependencies (per requirements.txt) , to reflect the new required version of zorp.

### CODE REVIEW

**Comment by amkwong** [comment]
Note: this has currently broken region view, I am fixing that right now

**Comment by abought** [comment]
Thanks! Let me know when it's ready for a look. Is there a time goal (like demo) in mind? Otherwise I might slate Monday to look at this.

**Comment by amkwong** [comment]
> Thanks! Let me know when it's ready for a look. Is there a time goal (like demo) in mind? Otherwise I might slate Monday to look at this.

I am aiming to have this functional on my local machine by tomorrow so I can show it at the meeting. Once I have something that works, then reviewing next Monday sounds good to me.

Thanks!

**Comment by amkwong** [comment]
The problem turned out to be a silly mistake that I should have caught earlier, and has been fixed accordingly. This PR is ready to be reviewed.

File: pheget/static/js/region.js [hunk 1]
```diff
@@ -3,6 +3,58 @@
 const API_BASE = 'https://portaldev.sph.umich.edu/api/v1/';
 // Set to smaller values for testing; go up to 50k or 200k after we make it more efficient
 const MAX_EXTENT = 500000;
+
+
+LocusZoom.TransformationFunctions.set('pip_yvalue', function (x) {return Math.max(Math.log10(x), -6);});
+
+var retrieveBySuffix = function(input, suffix) {
```
Comment (abought) [inline pheget/static/js/region.js]: I've refactored this function and moved duplicate transformation functions to common.js.

Oddly, the duplicate code was implemented in very slightly different ways across region vs variant.js. (eg whether it checked for null values; it wasn't clear to me if these values were allowed to be null)

@pjvandehaar , feel free to make adjustments if I missed an edge case.

**Comment by amkwong** [comment]
> * Have we evaluated the performance of the PIP lookup on a dense region (eg 500kb, our max allowed extent)? What is the worst case scenario for the amount of data involved for lookup + serialization?

I only tested this on my own laptop using the full data file, and I didn't look for the region of highest density, but the average lookup time was about a second to load the data in both the single-variant and region views.

**Comment by amkwong** [comment]
I forgot to remove custom ticks in the region plot for non-PIP views, so I fixed that. After updating zorp everything seems to work on my computer.

**Pull request merged**